CMakeLists.txt

```diff
@@ -316,6 +316,15 @@ endif()
 if (MFEM_USE_RAJA)
    find_package(RAJA REQUIRED)
 endif()
+   
+# UMPIRE
+if (MFEM_USE_UMPIRE)
+   find_package(UMPIRE REQUIRED)
+   set(MFEM_UMPIRE_HOST "mfem_host")
+    message(STATUS "MFEM: Umpire host allocator name: ${MFEM_UMPIRE_HOST}")
+   set(MFEM_UMPIRE_DEVICE "mfem_device")
+    message(STATUS "MFEM: Umpire device allocator name: ${MFEM_UMPIRE_DEVICE}")
```

Comment:
Do we really need these config variables? These strings can be configured at run-time with `MemoryManager::SetUmpireAllocatorNames()`, right?

Reply:
That was something we talked about with the `Umpire` team, I remember they wanted such a way to set the two names at compile time.
We can asked how the application now envisage using it.

Reply:
The run-time option is clearly more flexible. I'm just trying to remove unnecessary complexity from the build systems.

Reply:
I removed the compile-time variables for Umpire.